These new tests in commit `b2b10a3` all pass consistently on my Macbook (10.14.6) within a fresh virtual environment with Python 3.7.1. The only ones that fail on my local machine are

```
============================================== FAILURES ===============================================
________________________________________ tests/linalg_test.py _________________________________________
[gw0] darwin -- Python 3.7.1 /Users/k28390/jax/venv/bin/python3
worker 'gw0' crashed while running 'tests/linalg_test.py::NumpyLinalgTest::testInv_shape=float32[200,200]'
________________________________________ tests/linalg_test.py _________________________________________
[gw1] darwin -- Python 3.7.1 /Users/k28390/jax/venv/bin/python3
worker 'gw1' crashed while running 'tests/linalg_test.py::ScipyLinalgTest::testLuFactor_n=complex64[200,200]'
```

which seemed unrelated.

--- a/jax/random.py
+++ b/jax/random.py
@@ -443,7 +443,7 @@ def _multivariate_normal(key, mean, cov, shape, dtype):
       shape = shape + cov.shape[:1]
       normal_samples = normal(key, shape, dtype)
       if cov.ndim == 2:
-        samples = np.tensordot(normal_samples, cholesky(cov), axes=(-1, 0))
+        samples = np.tensordot(normal_samples, cholesky(cov), axes=1)
       else:
         samples = normal_samples * np.sqrt(cov)
   else:

--- a/jax/random.py
+++ b/jax/random.py
@@ -449,8 +449,6 @@ def _multivariate_normal(key, mean, cov, shape, dtype):
   else:
     if hasattr(mean, "shape") and mean.ndim > 0:
       shape = shape + mean.shape[:1]
-    else:
-      shape = shape
     normal_samples = normal(key, shape, dtype)
     samples = np.sqrt(cov) * normal_samples
   return samples + mean

--- a/jax/random.py
+++ b/jax/random.py
@@ -443,7 +443,7 @@ def _multivariate_normal(key, mean, cov, shape, dtype):
       shape = shape + cov.shape[:1]
       normal_samples = normal(key, shape, dtype)
       if cov.ndim == 2:
-        samples = np.tensordot(normal_samples, cholesky(cov), axes=1)
+        samples = np.tensordot(normal_samples, cholesky(cov), axes=(-1, 0))
       else:
         samples = normal_samples * np.sqrt(cov)
   else:

--- a/tests/random_test.py
+++ b/tests/random_test.py
@@ -384,7 +384,7 @@ class LaxRandomTest(jtu.JaxTestCase):
     compiled_samples = crand(key, mean, cov)
     if hasattr(cov, "shape") and cov.ndim == 2:
       inv_scale = scipy.linalg.lapack.dtrtri(onp.linalg.cholesky(cov), lower=True)[0]
-      rescale = lambda x: x.dot(inv_scale)
+      rescale = lambda x: onp.tensordot(x, inv_scale, axes=(-1, 1))
     else:
       rescale = lambda x: x / np.sqrt(cov)
     for samples in [uncompiled_samples, compiled_samples]:

--- a/tests/random_test.py
+++ b/tests/random_test.py
@@ -368,12 +368,12 @@ class LaxRandomTest(jtu.JaxTestCase):
 
   @parameterized.named_parameters(jtu.cases_from_list(
       {"testcase_name": "_mean={}_cov={}_{}".format(mean, cov, dtype),
-          "mean": mean, "cov": cov, "dtype": dtype}
+        "mean": mean, "cov": cov, "dtype": dtype}
       for mean in [0, 5, np.asarray([1, 2, 3])]
       for cov in [.1, 5, np.asarray([4, 5, 6]),
-          np.asarray([[4.60, 2.86, 2.33],
-          [2.86, 3.04, 1.74],
-          [2.33, 1.74, 1.83]])]
+        np.asarray([[4.60, 2.86, 2.33],
+        [2.86, 3.04, 1.74],
+        [2.33, 1.74, 1.83]])]
       for dtype in [onp.float32, onp.float64]))
   def testMultivariateNormal(self, mean, cov, dtype):
     key = random.PRNGKey(0)
@@ -388,9 +388,9 @@ class LaxRandomTest(jtu.JaxTestCase):
     else:
       rescale = lambda x: x / np.sqrt(cov)
     for samples in [uncompiled_samples, compiled_samples]:
-        self._CheckKolmogorovSmirnovCDF(
-                rescale(samples - mean).reshape(-1),
-                scipy.stats.norm().cdf)
+      self._CheckKolmogorovSmirnovCDF(
+          rescale(samples - mean).reshape(-1),
+          scipy.stats.norm().cdf)
 
   def testIssue222(self):
     x = random.randint(random.PRNGKey(10003), (), 0, 0)

